tasks/Makefile.hugo

```diff
@@ -10,6 +10,5 @@ hugo/run: components/build
 hugo/build: components/build
 	@[ "$(HUGO_PUBLISH_DIR)" != "/" ] || (echo "Invalid HUGO_PUBLISH_DIR=$(HUGO_PUBLISH_DIR)"; exit 1)
 	rm -rf $(HUGO_PUBLISH_DIR)
-	$(DOCKER_RUN) hugo --templateMetrics --stepAnalysis --config $(HUGO_CONFIG)
-
+	$(DOCKER_RUN) hugo --templateMetrics --trace /dev/stdout --config $(HUGO_CONFIG)
```

Comment:
```suggestion
	$(DOCKER_RUN) hugo --templateMetrics --config $(HUGO_CONFIG)
```

The ` --trace /dev/stdout ` is not the same as `--stepAnalysis` (is my guess) because this led to binary output to `stdout`

Reply:
From https://github.com/gohugoio/hugo/issues/5681:
```
--stepAnalysis is removed. If you want to really understand the latency in your project in Hugo, try the new --trace flag and pass that file to the many tools that read Go Trace files. There are also some newly hidden flags in --profile-cpu, --profile-men, --profile-mutex, hidden because they are considered to be only of interest to developers.
```

Reply:
@osterman So no `--trace` outputting?

Reply:
Not unless there's a way to format it in non-binary ;)

Reply:
let's skip for now.